RHDEVDOCS-2108: Installing OpenShift Pipelines Operator Using CLI is unclear (#83755)
openshift/openshift-docs: modules/op-installing-pipelines-operator-using-the-cli.adoc

// * install_config/installing-pipelines.adoc

:_mod-docs-content-type: PROCEDURE

> [id="op-installing-pipelines-operator-by-using-the-cli_{context}"]

Reviewer: ```suggestion
[id="op-installing-pipelines-operator-using-the-cli_{context}"]
```
I suggest keeping this as it was for this reason: ID needs to match the name of the file, and that would require additional change around, changing links etc. But the heading can slightly differ from the ID, so I would suggest keeping the new heading, but leaving the ID as it was before

:_mod-docs-content-type: PROCEDURE
[id="op-installing-pipelines-operator-using-the-cli_{context}"]
= Installing the {pipelines-shortname} Operator by using the CLI

You can install {pipelines-title} Operator from the OperatorHub by using the CLI.

.Procedure

> . Create a Subscription object YAML file to subscribe a namespace to the {pipelines-title} Operator,

Reviewer: ```suggestion
. Create a `Subscription` object YAML file to subscribe a namespace to the {pipelines-title} Operator,
```

Reviewer: @shivanisathe25 Found this one more small change. Othervise, LGTM! :slightly_smiling_face:

:_mod-docs-content-type: PROCEDURE
[id="op-installing-pipelines-operator-using-the-cli_{context}"]
= Installing the {pipelines-shortname} Operator by using the CLI

> You can install {pipelines-title} Operator from the OperatorHub by using the CLI.

Reviewer 2: ```suggestion
You can install {pipelines-title} Operator from OperatorHub by using the command-line interface (CLI).
```

- Remove `the`: I see more search results in repo for "OperatorHub" than "the OperatorHub".
- Expand acronym (CLI) for its first occurrence in the module.

.Procedure

> . Create a `Subscription` object YAML file to subscribe a namespace to the {pipelines-title} Operator,
> for example, `sub.yaml`:

Reviewer 2: ```suggestion
. Create a `Subscription` object YAML file to subscribe a namespace to the {pipelines-title} Operator, for example, `sub.yaml`:
```

+

> .Example Subscription

Reviewer 2: ```suggestion
.Example `Subscription` YAML
```

  name: openshift-pipelines-operator
  namespace: openshift-operators
spec:

>   channel:  <channel name> # <1>

Reviewer: ```suggestion
  channel:  <channel_name> # <1>
```

  name: openshift-pipelines-operator-rh # <2>
  source: redhat-operators # <3>
  sourceNamespace: openshift-marketplace <4>
----

> <1> Name of the channel to subscribe to. The `pipelines-<version>` channel is the default channel. For example, the default channel for {pipelines-title} Operator version `1.7` is `pipelines-1.7`. The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator.

Reviewer 2: ```suggestion
<1> Name of the channel that you want to subscribe. The `pipelines-<version>` channel is the default channel. For example, the default channel for {pipelines-title} Operator version `1.7` is `pipelines-1.7`. The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator.
```

<2> Name of the Operator to subscribe to.

> <3> Name of the CatalogSource that provides the Operator.

Reviewer: ```suggestion
<3> Name of the `CatalogSource` that the Operator provides .
```

> <3> Name of the `CatalogSource` that the Operator provides.

Reviewer 2: ```suggestion
<3> Name of the `CatalogSource` object that the Operator provides.
```

> <4> Namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.

Reviewer: ```suggestion
<4> Namespace of the `CatalogSource`. Use `openshift-marketplace` for the default OperatorHub `CatalogSource` resources.
```

> <4> Namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.

Reviewer 2: ```suggestion
<4> Namespace of the `CatalogSource` object. Use `openshift-marketplace` for the default OperatorHub catalog sources.
```

I know this is probably out of scope for this PR. But it would be best if you can consider making this change.

> . Create the `Subscription` object:

Reviewer 2: ```suggestion
. Create the `Subscription` object by running the following command:
```

> . Create the `Subscription` object:

Reviewer 2: Add [source,terminal] on line no 36

>   source: redhat-operators <3>

Reviewer: For the module title, from:
`= Installing the {pipelines-shortname} Operator using the CLI`
to this (to erase double meaning)
`= Installing the {pipelines-shortname} Operator by using the CLI`

>   source: redhat-operators <3>

Reviewer: from:
`channel:  <channel name> <1>`
to:
`channel:  <channel _name> # <1>`

>   source: redhat-operators <3>

Reviewer: from:
`name: openshift-pipelines-operator-rh <2>`
to:
`name: openshift-pipelines-operator-rh # <2>`

>   source: redhat-operators <3>

Reviewer: ```suggestion
  source: redhat-operators # <3>
```

>   source: redhat-operators <3>

Reviewer: from:
`. Create a Subscription object YAML file to subscribe...`
to:
```
. Create a `Subscription` object YAML file to subscribe...
```

>   source: redhat-operators <3>

Reviewer: remove `[discrete]` line before the `.procedure`

>   source: redhat-operators <3>

Reviewer: from:
`You can install {pipelines-title} Operator from the OperatorHub using the CLI.`
to:
`You can install {pipelines-title} Operator from the OperatorHub by using the CLI.`

  sourceNamespace: openshift-marketplace <4>
----
<1> Name of the channel to subscribe to. The `pipelines-<version>` channel is the default channel. For example, the default channel for {pipelines-title} Operator version `1.7` is `pipelines-1.7`. The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator.
<2> Name of the Operator to subscribe to.
<3> Name of the CatalogSource that provides the Operator.

> <4> Namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.

Reviewer: from:
`. Create the Subscription object:`
to:
```
. Create the `Subscription` object:
```

Reviewer: It is a line 34 :slightly_smiling_face:
